daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/archive/DecodeV1Spec.scala

```diff
@@ -1062,6 +1067,7 @@ class DecodeV1Spec
                 Map(),
                 EPrimCon(PCUnit),
                 Map(),
+                TBuiltin(BTUnit),
```

Comment:
```suggestion
                TUnit,
```

defined in `com.daml.lf.language.Util`